mk/cfg/x86_64-unknown-linux-uclibc.mk

```diff
@@ -0,0 +1,29 @@
+# x86_64-unknown-linux-uclibc configuration
```

Comment:
> So, is your decision to preference using rustbuild only?

The Makefiles are going to be deprecated at some point (hopefully soon! I'm tired of having to test my PRs on and adapt them to work with two build systems -- it's time consuming), and supporting one build system means less testing on our part.

> Because I need support for using rust with buildroot

But if you need it then I'm not going to oppose. By the way, could you point me to this "rust with buildroot" project/repository? It seems that they are building on top of the old Makefile-based build system, but they should move to rustbuild at some point.